### PR TITLE
Add Marker Module

### PR DESCRIPTION
**Pull request by sarahcodes100**
Adds a new module for demonstrating dynamic marker APIs

Demonstrates:
- Adding markers
- Toggling marker visibility
- Adding point markers
- Adding bitmap markers
- Adding polyline markers
- Adding polygon markers
- Removing markers

Depends on: https://github.com/tangrams/tangram-es/pull/1056

### CODE REVIEW

File: dynamicmarkers/build.gradle [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    sourceSets.main.assets.srcDirs = ['../styles']
+}
+
+repositories {
+    mavenLocal()
+}
+
+dependencies {
+    compile 'com.android.support:appcompat-v7:24.2.1'
+    compile 'com.mapzen.tangram:tangram:0.4.4-SNAPSHOT'
```
Comment (tallytalwar) [inline dynamicmarkers/build.gradle]: This needs updating now.

Comment (tallytalwar) [inline dynamicmarkers/build.gradle]: This needs updating now.

**Comment by tallytalwar** [comment]
Sorry for not reviewing this uptill now.

All this code looks good. But seems like scene file is not being loaded and hence no map gets displayed. Probably to do with the updates of handling the Urls in tangram.

**Review by tallytalwar — CHANGES_REQUESTED** [review]

File: dynamicmarkers/build.gradle (hunk 1 above)
Comment (sarahcodes100) [reply dynamicmarkers/build.gradle]: Updated to 0.4.6!

**Pull request merged**